integrasjon/pdl-klient/src/main/java/no/nav/vedtak/felles/integrasjon/pdl/PdlKlientMedCache.java

```diff
@@ -91,17 +137,10 @@ public PdlKlientMedCache(PdlKlient pdlKlient, int cacheSize, long cacheTimeoutMi
 
         Identliste identliste = pdlKlient.hentIdenter(request, projeksjon, tema);
 
-        return identliste.getIdenter().stream().filter(s -> s.getGruppe().equals(identGruppe)).findFirst().map(i -> i.getIdent());
-    }
-
-    public Set<String> hentAktørIdForPersonIdentSet(Set<String> personIdentSet) {
-
-
-
-
-        return Collections.emptySet();
+        return identliste.getIdenter().stream().filter(s -> s.getGruppe().equals(identGruppe)).findFirst().map(IdentInformasjon::getIdent);
     }
 
+    //TODO Trenger vi denne, er ekvivalenten i bruk noe sted Jens-Otto?
```

Comment:
Kan du svare meg på dette @jolarsen ?

Reply:
Ekvivalenten var i bruk i PdpRequestBuilder for å transformere en mengde identer til aktørid.
Det er få steder vi har abac-attributtype personident/fnr - så jeg har kalt sekvensielt i fpsak
Bolk-spørring krever at man spør med Authorization systembrukertoken - så da må man evt snekre noen spesielt i Oidc-klienten - for nå forwardes klient-token

Reply:
Da tolker jeg dette til at vi trenger ikke denne lenger nå. Er dette riktig forstått?

Reply:
FP trenger den ikke.